### PR TITLE
DEV-3403-new go exercise get alpha

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

File: subjects/get-alpha/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## getalpha
+
+### Instructions
+
+Write  a program that takes an `int` [0-127] as an argument and returns the corresponding ASCII table character, followed by newline.
+
+- If the integer above 127 or less than 0 returns newline.
+- If it's not a number returns newline.
+- If the number of arguments is above 1 return newline.
```
Comment (tiagocollot) [inline subjects/get-alpha/README.md:9]: Needs to be fixed. "...is above 1  return a newline `\n`."

File: subjects/get-alpha/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+## getalpha
+
```
[the remark this answers is not in the document]
Comment (jotapero) [reply subjects/get-alpha/README.md:2]: fixed

File: subjects/get-alpha/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- If the number of arguments is above 1 return newline.
+
+### Usage
+
+
+```go
+$ go run . | cat -e
+$
+$ go run . "98" | cat -e
+a
```
[the remark this answers is not in the document]
Comment (jotapero) [reply subjects/get-alpha/README.md]: fixed

File: subjects/get-alpha/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+$ go run . "95" | cat -e
+;
```
[the remark this answers is not in the document]
Comment (jotapero) [reply subjects/get-alpha/README.md]: fixed

File: subjects/get-alpha/README.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+## getalpha
+
+### Instructions
+
+Write  a program that takes an `int` [0-127] as an argument and returns the corresponding ASCII table character, followed by newline.
```
[the remark this answers is not in the document]
Comment (jotapero) [reply subjects/get-alpha/README.md]: fixed

File: subjects/get-alpha/README.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+- If the integer above 127 or less than 0 returns newline.
```
[the remark this answers is not in the document]
Comment (jotapero) [reply subjects/get-alpha/README.md]: fixed

File: subjects/get-alpha/README.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+- If it's not a number returns newline.
```
[the remark this answers is not in the document]
Comment (jotapero) [reply subjects/get-alpha/README.md]: fixed

File: subjects/get-alpha/README.md (hunk 1 above)
Comment (jotapero) [reply subjects/get-alpha/README.md]: fixed

File: subjects/get-alpha/README.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+```go
+$ go run . | cat -e
+$
+$ go run . "98" | cat -e
+b
+$ go run . "95" | cat -e
+_
+$ go run . "95"  "102" | cat -e
+$
+```
```
Comment (eslopfer) [inline subjects/get-alpha/README.md:23]: Please run prettier

**Review by eslopfer — DISMISSED** [review]

**Pull request merged**